platform/extract-apt-setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e -u
+
+cd "$(dirname "$0")"
```

Comment:
If this is always run in the deploy chroot, you could just use absolute paths. Also, if this is a utility script, why not add `bazel build //apt-setup:package.deb` to it too?

Reply:
this is run in the build chroot.  I guess I could add that

Reply:
added